contrib/cirrus/lib.sh

```diff
@@ -151,16 +165,118 @@ install_ooe() {
 }
 
 showrun() {
-    if [[ "$1" == "--background" ]]
+    local context
+    context=($(caller 0))
+    echo '+ '$(printf " %q" "$@")"  # ${context[2]}:${context[0]} in ${context[1]}()" > /dev/stderr
+    "$@"
+}
+
+comment_out_storage_mountopt() {
+    local FILEPATH=/etc/containers/storage.conf
+    echo ">>>>>"
+    echo ">>>>> Warning: comment_out_storage_mountopt() is modifying $FILEPATH"
+    echo ">>>>>"
+    sed -i -r -e 's/^(mountopt = .+)/#\1/' $FILEPATH
+}
+
+in_podman() {
+    req_env_var IN_PODMAN_NAME GOSRC HOME OS_RELEASE_ID
+    [[ -n "$@" ]] || \
+        die 7 "Must specify FQIN and command with arguments to execute"
+    local envargs
+    local envname
+    local envvalue
+    local envrx='(^CIRRUS_.+)|(^BUILDAH_+)|(^STOTAGE_)|(^CI$)|(^CROSS_TARGET$)|(^IN_PODMAN_.+)'
```

Comment:
`STOTAGE` looks like a typo

Reply:
Ooohhh, excellent catch.  It is indeed a literal typo.